src/test/java/io/aiven/kafka/connect/opensearch/OpensearchSinkTaskTest.java

```diff
@@ -51,4 +70,22 @@ void convertTopicToIndexName() {
 
     }
 
+    @Test
+    void failToStartWhenReportIsConfiguredAndErrantRecordReporterIsMissing(final @Mock SinkTaskContext context) {
```

Comment:
Whoa! Didn't realize you could use `@Mock` objects as method parameters for test cases. Super cool!